Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -140,8 +140,11 @@ public struct TuistCommand: AsyncParsableCommand {
         ]
         let exitCode = exitCode(for: error).rawValue
 
-        if let clientError = error as? ClientError,
-           let underlyingServerClientError = clientError.underlyingError as? ServerClientAuthenticationError
+        if error.localizedDescription.contains("ArgumentParser") {
```

Comment:
Unfortunately `ArgumentParser` errors are private, so I have to rely on string-based checks.